### PR TITLE
YLP-298 Enable Spanish and Italian languages

### PR DESCRIPTION
**Pull request by yvon-dblg**
PR need to be updated after https://github.com/mdblp/translations/pull/5 is merged

### CODE REVIEW

File: artifact-lang.sh [hunk 1]
```diff
@@ -3,14 +3,14 @@ set -eu
 
 function retrieveLanguageParameters() {
   declare -a languages
-  languages=(en fr de nl)
+  languages=(de en es fr it nl)
```
Comment (franck-fourel) [inline artifact-lang.sh:6]: can we remove this dependency and get the list from the json file as below
```suggestion
  languages=(`jq --raw-output '.resources | keys | @tsv' locales/languages.json`)
```
if we use jq, we have to be sure it's installed when running in jenkins

**Review by franck-fourel — CHANGES_REQUESTED** [review]
one request to remove the update on artifact-lang.sh and get the list of languages from json file

File: artifact-lang.sh (hunk 1 above)
Comment (yvon-dblg) [reply artifact-lang.sh:6]: Or without jq:
```bash
echo 'const l=require("./locales/languages.json"); console.log(Object.getOwnPropertyNames(l.resources).join(" "));' | node -
```

Comment (franck-fourel) [reply artifact-lang.sh:6]: no more dependency, great !!!

**Review by franck-fourel — APPROVED** [review]

**Pull request merged**